pages.es/linux/userdel.md

```diff
@@ -1,21 +1,17 @@
 # userdel
 
 > Elimina una cuenta de usuario o elimina un usuario de un grupo.
-> Nota: todos los comandos deben ser ejecutados como root.
+> Ver también: `users`, `useradd`, `usermod`.
 > Más información: <https://manned.org/userdel>.
 
 - Elimina un usuario:
 
-`userdel {{nombre}}`
+`sudo userdel {{nombre}}`
 
-- Elimina un usuario junto con su directorio home y mail spool:
+- Elimina un usuario en otro directorio raíz(root):
```

Comment:
```suggestion
- Elimina un usuario en otro directorio raíz:
```

Reply:
The original in english reads "Remove a user along with the home directory and mail spool:", I miss the mail info, maybe change "raíz" for "personal" ?

Elimina un usuario con su directorio personal y archivos de correo.

Reply:
Elimina un usuario con su directorio personal y archivos de correo. LGTM